components/SEOBundle/bundle/Core/MetaNameSchema.php

```diff
@@ -112,19 +112,13 @@ public function setImageVariationService(VariationHandler $handler): void
         $this->imageVariationService = $handler;
     }
 
-    public function resolveMeta(Meta $meta, Content $content, ContentType $contentType = null): bool
```

Comment:
Can we keep it an mark it deprecated? Like just a comment on top of the function. So we will remove in the next version

Reply:
@Plopix done!